### DISCUSSION TITLE
docker containers do not start

### DISCUSSION THREAD

**Discussion by proton1k** [supabase/supabase#9074] [Questions]
Such commands as `supabase start`, `stop` and `status` are hanging and do not show any warnings or error messages. Docker installation might be broken on Apple chip M1 Macbook, but can't find the exact issue. There's a good request to support podman in https://github.com/supabase/cli/issues/265 but so far does not work as a replacement to docker.

What should be the logic to troubleshoot the local containers running in this case? (no output from CLI at all)

**Comment by adrianstamp** [comment]
Same issue here, have been trying for 2 days to get `supabase start` working, every time I restart docker and rerun I get a different error. but most of the time it just hangs. I have currently been waiting 8 hours to "restore branches", but often hangs while starting the database.

<img width="389" alt="image" src="https://user-images.githubusercontent.com/65253922/192309360-d9edb4ab-5bb3-4f93-9afb-d1a290f38ad0.png">

Any debugging help as o1dnik asked for would be very useful!

**Comment by atuttle** [comment]
Anyone ever figure this out? I just ran into this tonight randomly.

**Reply by adrianstamp** [reply]
I don't fully remember but I think it was an issue with the version of docker. Try upgrading docker and rebuilding the image

**Reply by atuttle** [reply]
I did just upgrade docker today. 🤔 

I was able to resolve the issue by opening docker desktop and deleting all existing supabase-relevant images and containers. I had one container that showed status exited. Once I deleted all of those, I was able to run `supabase start` and it re-downloaded everything. For the moment, it seems like I'm back in business...

**Reply by adrianstamp** [reply]
Ok great, it's a couple of years since I've had the issue, but it seems like it doesn't play nicely with certain versions of docker

**Comment by oscadev** [comment]
I had my docker desktop RAM limit set to 2GB. I raised it to 8GB and it started working, and I noticed a new instance uses about 2.5GB.
